docs/consume-packages/Package-References-in-Project-Files.md

```diff
@@ -189,9 +189,9 @@ If this property is set, NuGet restore will generate a lock file - `packages.loc
 ### `restore` behavior with lock file
 If a lock file is present for project, NuGet uses this lock file to run `restore`. NuGet does a quick check to see if there were any changes in the package dependencies as mentioned in the project file (or dependent projects' files) and if there were no changes it just restores the packages mentioned in the lock file. There is no re-evaluation of package dependencies.
 
-If NuGet detects a change in the defined dependenices as mentioned in the project file(s), it re-evaluates the package graph and updates the lock file to reflect the new package closure for the project.
+If NuGet detects a change in the defined dependencies as mentioned in the project file(s), it re-evaluates the package graph and updates the lock file to reflect the new package closure for the project.
 
-For CI/CD and other scenarios, where you would not want to change the package dependenies on the fly, you can do so by setting the `lockedmode` to `true`:
+For CI/CD and other scenarios, where you would not want to change the package dependencies on the fly, you can do so by setting the `lockedmode` to `true`:
```

Comment:
Should `lockedmode` be `locked-mode`?  The example command below is: `dotnet.exe restore --locked-mode`

Reply:
@jainaashish

Reply:
Is `lockedmode` something that can be set anywhere? 

Can we reword it to say "enabling locked mode"? 
Because you need to do a different thing in dotnet.exe/ if running throguh msbuild.